dist/README.md

```diff
@@ -0,0 +1,83 @@
+# ngAddToCalendar
```

Comment:
All this stuff in the `/dist` folder is created when we run the build right? So could we add `dist/` to the .gitignore? Or do we want this kept in here?